html/html-quiz.md

```diff
@@ -1388,5 +1387,5 @@ As Steve Krug once said, happy talk must die.
       ![B](images/Q84-2.jpg)
 - [ ] C
       ![C](images/Q84-3.jpg)
-- [ ] D
+- [X] D
```

Comment:
B is correct here. (Show only "> Details"). Tested in both Firefox and Chrome

Reply:
@Muhammad-Osama-Qureshi thanks for your input and could you please take a look at @ruslanbes suggestion
would be happy to merge your PR
have a great day!